internal/contour/holdoff.go

```diff
@@ -42,16 +42,36 @@ type HoldoffNotifier struct {
 	mu    sync.Mutex
 	timer *time.Timer
 	last  time.Time
+
+	holdoffDelay    time.Duration
+	holdoffMaxDelay time.Duration
+}
+
+func (hn *HoldoffNotifier) SetHoldoffDelay(t time.Duration) {
+	hn.holdoffDelay = t
+}
+
+func (hn *HoldoffNotifier) SetHoldoffMaxDelay(t time.Duration) {
```

Comment:
What happens if these functions are called after the notifier is started? Its either going to be a data race or the updated values will be ignored.